src/central-submissions.rst

```diff
@@ -143,3 +147,52 @@ To connect with Excel or Power BI, follow these steps.
+| Received   | System      | The default state for all incoming submissions, assigned automatically by Central |
++------------+-------------+-----------------------------------------------------------------------------------+
+| Edited     | System      | Automatically assigned by Central whenever a submission is edited by any user     |
++------------+-------------+-----------------------------------------------------------------------------------+
+| Has Issues | User        | Can be assigned by project staff if a submission has problems                     |
++------------+-------------+-----------------------------------------------------------------------------------+
+| Approved   | User        | Can be assigned by project staff to approve a submission                          |
++------------+-------------+-----------------------------------------------------------------------------------+
+| Rejected   | User        | Can be assigned by project staff to reject a submission                           |
++------------+-------------+-----------------------------------------------------------------------------------+
+
+The ``Received`` and ``Edited`` states are automatically set by Central any time a submission is uploaded or edited. The other states are assigned by project staff. Review states have suggested meanings, but *they do not do anything by themselves*. You are free to use these states however you'd like.
+
+Once submissions have been reviewed, the submission table download and the OData connection both allow submissions to be filtered by review state. This lets you, for example, download only all the approved submissions.
+
+.. _central-submissions-details:
+
+Submission Details
+------------------
+
+As of version 1.2, each submission has its own detail page which provides basic information about the submission, an activity history of action and discussion on that submission, and tools for updating the submission review state and data itself.
+
+   .. image:: /img/central-submissions/details.png
+
+The title at the top is pulled from the ``instanceName`` metadata tag if there is one, otherwise it will be the ``instanceID``. You can set up your form to compute an ``instanceName`` based on the data in each submission. You may want to do this if you plan on using this page a lot, because it makes navigation much easier to have friendly names at the top of the page and in the web browser title and tab.
+
+Basic detail can be found along the left. If there are expected media attachments for this submission, that status information will be provided.
+
+The main activity feed on the right shows you the discussion and action history of the submission. Any review state changes, comments, and edits will appear here. At the top of the activity feed, you can :guilabel:`Review` a submission to assign a new review state, :guilabel:`Edit` the submission directly using Enketo in your web browser, or type in the box to begin leaving a comment.
```

Comment:
s/Enketo in//

Reply:
i've been adding this bc in the past @lognaturel has recommended me to, because there are differences between enketo and collect.